### DISCUSSION TITLE
Allow to get the Geolocation

### DISCUSSION THREAD

**Discussion by Uno-count** [seleniumbase/SeleniumBase#3307] [Q&A] [answered]
I need to allow location access for the website I'm automating, but I'm unsure of the best way to configure the browser for that. Could anyone share their experience or suggest the most effective approach for enabling location permissions when running tests?

**Comment by mdmintz** [comment, marked as answer]
https://github.com/seleniumbase/SeleniumBase/blob/master/examples/test_geolocation.py
